docs/user/cli/export.rst

```diff
@@ -111,8 +122,11 @@ The ``--output-format`` argument defines the encoding used for export. Currently
     map generation it is highly recommended to use GeoServer. Additionally, the resulting file from this command
     requires the use of an online browser to open in order to access online resources.
 
-The ``--output`` argument can be used to export to a file. By default, export data is written to the standard
-output stream.
+The ``--output`` argument can be used to export to a file. By default, export data is written to standard output.
+
+The ``--sort-by`` argument can be used to sort the output by one or more attributes. Note that for local exports,
```

Comment:
Is the implication that sorting for distributed exports work well?:)

Reply:
I'm not sure what you mean by 'well', but it works :)